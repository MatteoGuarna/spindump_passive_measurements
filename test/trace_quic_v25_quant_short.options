--textual --format json --report-spins --report-notes
