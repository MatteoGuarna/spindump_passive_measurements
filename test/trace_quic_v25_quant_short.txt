A simple draft-25 trace with Quant, as well as ICMP and ICMPv6 tests to quant.eggert.org
