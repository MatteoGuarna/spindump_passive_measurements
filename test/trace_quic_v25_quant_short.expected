[
{ "Event": "new", "Type": "QUIC", "Addrs": ["100.72.46.172","13.48.145.10"], "Session": "0497ee9c-c4530173851bc6a574f4c16bb857c206 (64377:4433)", "Ts": 1580731950933519, "State": "Starting", "Notes": "No rsp,v.qn25,0-RTT,no spin", "Packets1": 0, "Packets2": 0, "Bytes1": 0, "Bytes2": 0 },
{ "Event": "change", "Type": "QUIC", "Addrs": ["100.72.46.172","13.48.145.10"], "Session": "0497ee9c-0481fe5f (64377:4433)", "Ts": 1580731950968485, "State": "Starting", "Notes": "No rsp,v.qn25,0-RTT,no spin", "Packets1": 1, "Packets2": 0, "Bytes1": 1228, "Bytes2": 0, "Bandwidth1": 1228, "Bandwidth2": 0 },
{ "Event": "measurement", "Type": "QUIC", "Addrs": ["100.72.46.172","13.48.145.10"], "Session": "0497ee9c-0481fe5f (64377:4433)", "Ts": 1580731950968485, "State": "Up", "Notes": "No rsp,v.qn25,0-RTT,no spin", "Right_rtt": 34966, "Packets1": 1, "Packets2": 0, "Bytes1": 1228, "Bytes2": 0, "Bandwidth1": 1228, "Bandwidth2": 0 },
{ "Event": "spinvalue", "Type": "QUIC", "Addrs": ["100.72.46.172","13.48.145.10"], "Session": "97ee9cfb-0481fe5f (64377:4433)", "Ts": 1580731951004418, "State": "Up", "Notes": "V.qn25,0-RTT,no spin", "Value": 0, "Who": "responder", "Packets1": 3, "Packets2": 2, "Bytes1": 2872, "Bytes2": 1570, "Bandwidth1": 2872, "Bandwidth2": 1570 },
{ "Event": "spinvalue", "Type": "QUIC", "Addrs": ["100.72.46.172","13.48.145.10"], "Session": "97ee9cfb-0481fe5f (64377:4433)", "Ts": 1580731951005586, "State": "Up", "Notes": "V.qn25,0-RTT,no spin", "Value": 0, "Who": "responder", "Packets1": 3, "Packets2": 3, "Bytes1": 2872, "Bytes2": 3070, "Bandwidth1": 2872, "Bandwidth2": 3070 },
{ "Event": "spinvalue", "Type": "QUIC", "Addrs": ["100.72.46.172","13.48.145.10"], "Session": "97ee9cfb-0481fe5f (64377:4433)", "Ts": 1580731951005949, "State": "Up", "Notes": "V.qn25,0-RTT,no spin", "Value": 1, "Who": "initiator", "Packets1": 3, "Packets2": 4, "Bytes1": 2872, "Bytes2": 3793, "Bandwidth1": 2872, "Bandwidth2": 3793 },
{ "Event": "spinvalue", "Type": "QUIC", "Addrs": ["100.72.46.172","13.48.145.10"], "Session": "97e9bd09-0481fe5f (64377:4433)", "Ts": 1580731951010035, "State": "Up", "Notes": "V.qn25,0-RTT,no spin", "Value": 1, "Who": "initiator", "Packets1": 4, "Packets2": 5, "Bytes1": 2929, "Bytes2": 3863, "Bandwidth1": 2929, "Bandwidth2": 3863 },
{ "Event": "spinvalue", "Type": "QUIC", "Addrs": ["100.72.46.172","13.48.145.10"], "Session": "97e9bd09-0481fe5f (64377:4433)", "Ts": 1580731951031003, "State": "Up", "Notes": "V.qn25,0-RTT,no spin", "Value": 0, "Who": "responder", "Packets1": 5, "Packets2": 5, "Bytes1": 2982, "Bytes2": 3863, "Bandwidth1": 2982, "Bandwidth2": 3863 },
{ "Event": "spinvalue", "Type": "QUIC", "Addrs": ["100.72.46.172","13.48.145.10"], "Session": "97e9bd09-0481fe5f (64377:4433)", "Ts": 1580731951042849, "State": "Up", "Notes": "V.qn25,0-RTT,no spin", "Value": 1, "Who": "responder", "Packets1": 5, "Packets2": 6, "Bytes1": 2982, "Bytes2": 3963, "Bandwidth1": 2982, "Bandwidth2": 3963 },
{ "Event": "new", "Type": "ICMP", "Addrs": ["2a00:79e1:abc:301:a8eb:bd46:55b0:1603","2a05:d016:e5c:ae01:ed56:9f43:327d:964b"], "Session": "62712", "Ts": 1580731955776821, "State": "Starting", "Notes": "No response", "Packets1": 0, "Packets2": 0, "Bytes1": 0, "Bytes2": 0 },
{ "Event": "measurement", "Type": "ICMP", "Addrs": ["2a00:79e1:abc:301:a8eb:bd46:55b0:1603","2a05:d016:e5c:ae01:ed56:9f43:327d:964b"], "Session": "62712", "Ts": 1580731955810975, "State": "Up", "Notes": "No response", "Right_rtt": 34154, "Packets1": 1, "Packets2": 0, "Bytes1": 56, "Bytes2": 0, "Bandwidth1": 56, "Bandwidth2": 0 },
{ "Event": "measurement", "Type": "ICMP", "Addrs": ["2a00:79e1:abc:301:a8eb:bd46:55b0:1603","2a05:d016:e5c:ae01:ed56:9f43:327d:964b"], "Session": "62712", "Ts": 1580731956814627, "State": "Up", "Right_rtt": 33931, "Packets1": 2, "Packets2": 1, "Bytes1": 112, "Bytes2": 56, "Bandwidth1": 56, "Bandwidth2": 56 },
{ "Event": "new", "Type": "ICMP", "Addrs": ["100.72.46.172","13.48.145.10"], "Session": "63733", "Ts": 1580731959166487, "State": "Starting", "Notes": "No response", "Packets1": 0, "Packets2": 0, "Bytes1": 0, "Bytes2": 0 },
{ "Event": "measurement", "Type": "ICMP", "Addrs": ["100.72.46.172","13.48.145.10"], "Session": "63733", "Ts": 1580731959542961, "State": "Up", "Notes": "No response", "Right_rtt": 376474, "Packets1": 1, "Packets2": 0, "Bytes1": 84, "Bytes2": 0, "Bandwidth1": 84, "Bandwidth2": 0 },
{ "Event": "measurement", "Type": "ICMP", "Addrs": ["100.72.46.172","13.48.145.10"], "Session": "63733", "Ts": 1580731961013292, "State": "Up", "Right_rtt": 848967, "Packets1": 2, "Packets2": 1, "Bytes1": 168, "Bytes2": 84, "Bandwidth1": 168, "Bandwidth2": 84 },
{ "Event": "measurement", "Type": "ICMP", "Addrs": ["100.72.46.172","13.48.145.10"], "Session": "63733", "Ts": 1580731961768169, "State": "Up", "Right_rtt": 603747, "Packets1": 3, "Packets2": 2, "Bytes1": 252, "Bytes2": 168, "Bandwidth1": 168, "Bandwidth2": 84 },
{ "Event": "measurement", "Type": "ICMP", "Addrs": ["100.72.46.172","13.48.145.10"], "Session": "63733", "Ts": 1580731962796215, "State": "Up", "Right_rtt": 628519, "Packets1": 4, "Packets2": 3, "Bytes1": 336, "Bytes2": 252, "Bandwidth1": 84, "Bandwidth2": 84 }
]
